### DISCUSSION TITLE
TavernAi connection.

### DISCUSSION THREAD

**Discussion by Ffffffmaa** [oobabooga/text-generation-webui#4668] [General]
Hello, i was using this with the integration of Tavernai through api, but today, November 19 i updated the oobabooga and suddenly i am unable to connect the tavernai, any suggestions about how i would be able to fix this?

**Comment by HiroseKoichi** [comment]
TextGen WebUI switched to using OpenAI's API standard, so you'll need to update SillyTavern so it can use it. The switch doesn't involve OpenAI in any way, and everything is still running locally; this is just a cleaner and standardized implementation of the API.

**Reply by Ffffffmaa** [reply]
Thank you very much
